### PR TITLE
global admin dashboard

### PR DESCRIPTION
**Pull request by snopoke**
This serves as a place to put global admin tools. Currently:

* basic charts
* Options to download usage data and whatsapp number list

In future this can be expanded with more features as needed.

Access to these pages is permitted to `staff` users.

Note: this also moves the Django Admin to `/django-admin/`

image

FYI @bderenzi

### CODE REVIEW

**Review by stephherbers — COMMENTED** [review]
changes look good, but curious why we chose staff vs admin to have easy access to downloading phone numbers? In my eyes, that is more sensitive information than just the usage data so maybe that can be an additional permission or restricted the downloading of data to admins? I know this isn't a big deal now with the number of people that are staff vs admin on projects, but just thinking about in the future if it were to scale

**Comment by snopoke** [comment]
> changes look good, but curious why we chose staff vs admin to have easy access to downloading phone numbers? In my eyes, that is more sensitive information than just the usage data so maybe that can be an additional permission or restricted the downloading of data to admins? I know this isn't a big deal now with the number of people that are staff vs admin on projects, but just thinking about in the future if it were to scale

Sorry for the confusion. 

`staff` refers to the Django `is_staff` attribute on users which is what is used to gate access to the Django Admin panel. This would only be set for specific global admin users.

Also this dashboard is not team specific, it is a global admin dashboard.

**Review by stephherbers — APPROVED** [review]

File: assets/javascript/admin-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+'use strict'
+import Chart from 'chart.js/auto'
```
Comment (SmittieC) [inline assets/javascript/admin-dashboard.js:2]: The other day I was thinking about trying out plotly / dash, [but for django](https://django-plotly-dash.readthedocs.io/en/latest/). I have worked with it before outside of django and it is pretty easy to build beautiful and powerful dashboards with graphs etc. Maybe also something to consider if we ever need to go bigger

Comment (SmittieC) [reply assets/javascript/admin-dashboard.js:2]: And the nice thing about it is that the code is mostly python

Comment (snopoke) [reply assets/javascript/admin-dashboard.js:2]: Looks interesting

**Pull request merged**